### ISSUE TITLE
Please provide releases!

### ISSUE THREAD

**Issue by DanishProgrammer2011** [viruscamp/luadec#92]
Providing releases can help us download it easier and make it .exe so it is executable!

**Comment by pierrelasse** [comment]
I have fun installing make on wsl and then gcc and then failing after all.......

**Comment by 2265290305** [comment]
compile by self,you are programmer,编译是最基础的技能

**Comment by erroreyes** [comment]
> compile by self,you are programmer,编译是最基础的技能

Instructions for compiling for Windows is not clear. 

Maybe you would like to be a friendly person, and in the spirit open source software, give Windows users some guidance? I'm sure people would appreciate it more than a "do it yourself" comment, and it may make you feel good to help the community. Otherwise your comment is meaningless, contributes nothing, and you could have used your time doing something else, right? It could have taken you probably the same amount of time to give a quick hint that it took you to make an unhelpful comment. Maybe all that was needed was to give an alternative to `make linux` for compiling on Windows.

Compiling is the most basic skill, as you say, and so is being helpful.

**Comment by sirrjohn** [comment]
i agree, add releases.. im not used to compiling this kind of stuff.

**Comment by jzfed** [comment]
Don't listen to that arrogant guy(@2265290305 )'s trash talk, this is the way it should be provided, or a lot of users will fall into gcc errors and waste a lot of time when all they need is a compiled file. This guy didn't just sit around and wonder where he learned his trash talk.

**Comment by smiRaphi** [comment]
Compiling this for windows is REALLY simple, literally just clone the repo + submodules and then open the .sln files and hit build, but here are compiled windows release builds for 5.1-3:

[luadec.zip](https://github.com/user-attachments/files/25974886/luadec.zip)
